x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -161,8 +161,9 @@ func (r *rttMonitor) addSample(rtt time.Duration) {
 
 	r.samples[r.offset] = rtt
 	r.offset = (r.offset + 1) % len(r.samples)
-	// Set the minRTT and 90th percentile RTT of all collected samples. Require at least 5 samples before
-	// setting these to prevent noisy samples on startup from artificially increasing RTT.
+	// Set the minRTT and 90th percentile RTT of all collected samples. Require at least 10 samples before
+	// setting these to prevent noisy samples on startup from artificially increasing RTT and to allow the
+	// calculation of a 90th percentile.
```

Comment:
I increased `minSamples` to 10, because the 90th percentile can only be calculated once there are >= 10 samples in the samples slice. Let me know if you think this will cause issues with `minRTT` calculation.